package.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+set -e
+
+#LOCAL_HELM_MOJALOOP_REPO_URI=${HELM_MOJALOOP_REPO_URI:-'https://docs.mojaloop.io/charts/repo'}
+LOCAL_HELM_MOJALOOP_REPO_URI=https://docs.mojaloop.io/charts/repo
+
+#
+# Script to Package all charts, and create an index.yaml in ./repo directory
+#
+
+trap 'echo "Command failed...exiting. Please fix me!"' ERR
+
+echo "Removing old charts..."
+find ./ -name "charts"| xargs rm -Rf
+
+mkdir -p ./repo
+
+if [ "$1" ]; then
+    declare -a charts=("$1")
+else
+    declare -a charts=(
+        mojaloop/example-backend
+        mojaloop/common
+        mojaloop/admin-api-svc
+        mojaloop/fspiop-transfer-api-svc
+        mojaloop/mojaloop
```

Comment:
Do we need to include the charts related to bof here?

Reply:
Yes we do.

Reply:
I added the BOF charts to the script files and pushed the commits to same PR